### PR TITLE
Update and fix `highest-rated-comment` style

### LINKED ISSUE #4529: `highest-rated-comment` has a red background in comment header
While looking at issues on GitHub, I noticed that some comments has a bright red background header, where it was dark green before. 
E.g. this comment
https://github.com/vercel/swr/issues/93#issuecomment-551110903
image

Refined GitHub seems to somehow change the css variables definition
image

When I disable Refined GitHub, it is back to being the default black header background.

### PR DESCRIPTION
**Pull request by 4086606**
Closes #4529. (CC @fregante)

## Test URLs

https://github.com/isaacs/github/issues/402#issuecomment-239936197
https://github.com/vercel/swr/issues/93#issuecomment-551110903

### Before
bugfix before

### After
bugfix after

### CODE REVIEW

File: source/features/highest-rated-comment.css [hunk 1]
```diff
@@ -1,5 +1,17 @@
+.rgh-highest-rated-comment {
+	border: 2px solid var(--color-text-success);
+}
+
+.rgh-highest-rated-comment.timeline-comment--caret::before {
+	background-color: var(--color-text-success);
+}
+
+.rgh-highest-rated-comment.timeline-comment--caret::after {
+	left: -6px;
```
Comment (fregante) [inline source/features/highest-rated-comment.css:10]: Can you leave a comment describing what this does? That is: "It makes the caret’s border 2px"

File: source/features/highest-rated-comment.css [hunk 2]
```diff
@@ -1,5 +1,17 @@
+.rgh-highest-rated-comment {
+	border: 2px solid var(--color-text-success);
```
Comment (fregante) [inline source/features/highest-rated-comment.css:2]: It's working in Firefox but not in Chrome for me

Screen Shot 11

File: source/features/highest-rated-comment.tsx [hunk 3]
```diff
@@ -54,7 +54,7 @@ function highlightBestComment(bestComment: Element): void {
 			className="d-flex flex-items-center text-green color-text-success mr-1 tooltipped tooltipped-n"
 			aria-label="This comment has the most positive reactions on this issue."
 		>
-			<CheckIcon/>
+			<CheckCircleFillIcon size='small'/>
```
Comment (fregante) [inline source/features/highest-rated-comment.tsx:57]: I don't think this works on our codebase. Also it doesn't seem to be required
```suggestion
			<CheckCircleFillIcon />
```

**Review by fregante — CHANGES_REQUESTED** [review]

File: source/features/highest-rated-comment.css (hunk 1 above)
Comment (4086606) [reply source/features/highest-rated-comment.css]: Resolved in 108aa74

File: source/features/highest-rated-comment.css (hunk 2 above)
Comment (4086606) [reply source/features/highest-rated-comment.css:2]: That's strange. I'll compile, test and get back to you

Comment (4086606) [reply source/features/highest-rated-comment.css]: Resolved in 2fb9338

**Comment by 4086606** [comment]
Tested in Chrome and Firefox btw

**Comment by fregante** [comment]
Thank you! In Chrome the link you provided doesn't highlights the comment and that style overrides our green border, while in Firefox it doesn't. Could you fix that too? I merged the PR because that's a minor issue for now

**Comment by 4086606** [comment]
I'm not sure I follow, screenshot?

**Pull request merged**